_data/validator-libraries-modern.yml

```diff
@@ -322,7 +309,7 @@
       url: https://json-everything.net
       date-draft: [2020-12, 2019-09]
       draft: [7, 6]
-      notes: Powered by JsonSchema.Net; server-side validation
+      notes: Powered by JsonSchema.Net in Blazor WASM for client-side validation
     - name: jsonschema.dev
```

Comment:
Looks like jschon.dev (2020-12) and "JSON Schema Validator" (2019-09) should move above jsonschema.dev.